File: src/main/java/io/curiousoft/ijudi/ordermanagement/model/PushHeading.java
package io.curiousoft.ijudi.ordermanagement.model;

import java.util.Objects;

public class PushHeading {

    String body;
    String title;
    String icon;

    public PushHeading() {
    }

    public PushHeading(String body, String title, String icon) {
        this.body = body;
        this.title = title;
        this.icon = icon;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getIcon() {
        return icon;
    }

    public void setIcon(String icon) {
        this.icon = icon;
    }

    @Override
    public boolean equals(Object o) {
        return o instanceof PushHeading && Objects.equals(body, ((PushHeading) o).getBody()) && Objects.equals(title,
                ((PushHeading) o).getTitle()) && Objects.equals(icon, ((PushHeading) o).getIcon());
    }
}

File: izinga-commons/src/main/kotlin/commons/model/PushHeading.kt
package io.curiousoft.izinga.commons.model

class PushHeading {
    var body: String? = null
    var title: String? = null
    var icon: String? = null

    constructor()
    constructor(body: String?, title: String?, icon: String?) {
        this.body = body
        this.title = title
        this.icon = icon
    }

    override fun equals(o: Any?): Boolean {
        return o is PushHeading && body == o.body && title == o.title && icon == o.icon
    }
}